packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -39,7 +39,6 @@ public class RNReactNativeGutenbergBridgeModule extends ReactContextBaseJavaModu
     private static final String EVENT_NAME_REQUEST_GET_HTML = "requestGetHtml";
     private static final String EVENT_NAME_UPDATE_HTML = "updateHtml";
     private static final String EVENT_NAME_UPDATE_TITLE = "setTitle";
-    private static final String EVENT_NAME_FOCUS_TITLE = "setFocusOnTitle";
```

Comment:
Similar to this change, we can do the same on iOS and remove the reference to the `setFocusOnTitle` event:

https://github.com/WordPress/gutenberg/blob/4f3b456bf84bc7e2373898d72028c6877261d003/packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift#L413